STYLE.md

```diff
@@ -0,0 +1,78 @@
+# Style Guide
+
+The eigenda repo consists majorly of Go and Solidity code, so we break it up into two sections:
+- [Go Style Guide](#go-style-guide)
+- [Solidity Style Guide](#solidity-style-guide)
+
+Our implicit rule is to defer to each language's respective style guides (see [resources](#resources) section). 
+However, some disagreements keep coming up in PRs, so we document our conventions here to prevent too much bike-shedding.
+This style guide is thus by definition perpetually a work in progress.
+
+## Go Style Guide
+
+This document outlines the coding standards and best practices for our Go projects.
+
+### Code Organization
```

Comment:
Consider adding a section about small *files*, in addition to small functions

Mega files that contain multiple structs are no good. If a struct is non-trivial, it ought to have a dedicated file